content/md/en/docs/maintain/monitor.md

```diff
@@ -4,59 +4,174 @@ description:
+You can also view statistics about the CPU and memory configuration across nodes in the network.
+
+To see the full set of statistics available, click **Statistics**.
+
+![View statistics across nodes](/media/images/docs/node-statistics.png)
+
+The statistics available include details about the hardware and software running on the nodes in the network, including:
+
+- Software version.
+- Operating system.
+- CPU architecture and model.
+- Number of physical CPU cores.
+- Total memory.
+- Whether the node is a virtual machine.
+- Linux distribution and kernel version.
+- CPU and memory speed.
+- Disk speed.
 
-You can use the `--telemetry-url` command-line option when you start a node to specify the telemetry server that you want to send telemetry data to.
+### Filter by chain
 
+By default, the telemetry dashboard displays information about Polkadot and provides quick links to a subset of other chains.
+To display information for other chains, click **More** and select a different chain. 
 
+![Select other chains](/media/images/docs/more-chains.png)
 
-Why it’s important 
+## Customize the monitoring stack
 
+The default telemetry dashboard provides visibility into node and chain operations without requiring you to host or configure any backend monitoring or front-end services.
+However, many projects choose to augment or replace the default telemetry server with their own backend server and front-end dashboards.
 
-What it reports
+In general, setting up your own telemetry server involves establishing monitoring and alerting policies for both **on-chain events** and individual **node operations**.
 
+### On-chain activity
 
-How it works diagram and description 
+You can monitor on-chain activity for specific events, such as transactions submitted from a certain address, a change to the current validator set.
+On-chain monitoring typically involves connecting to RPC nodes to check for specific values, identify processing delays, or track the timing of events. 
+In most cases, you only need two RPC instances to handle requests for all of your hosts. 
+However, it's recommended that you run your own RPC servers to service these requests in case there are issues with the public RPC nodes. 
+Some examples of applications that query on-chain information are [polkabot](https://gitlab.com/Polkabot/polkabot) and [polkadot-basic-notification](https://github.com/paritytech/polkadot-basic-notification).
 
+### Node operations
 
-Enabling the default backend and front end services for a node
+You should monitor each node that you run on the network for basic information about its operation such as the current block height, the number of peer-to-peer connections, CPU usage, and the available free memory.
+By default, Substrate exposes many useful metrics on the http://localhost:9615/metrics endpoint. 
```

Comment:
This localhost endpoint is only available if a substrate node is running locally of course